File: M4/Proyecto01/app/src/main/java/com/example/proyecto01/MainActivity2.java
package com.example.proyecto01;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;


public class MainActivity2 extends AppCompatActivity {

    Button btnSumar;
    EditText txtNumero1;
    EditText txtNumero2;
    TextView lblResultado;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        btnSumar = findViewById(R.id.btnSumar);

        btnSumar.setOnClickListener(view -> {

            Toast.makeText(this, "Presionó el botón sumar", Toast.LENGTH_LONG).show();

        });
    }
}

File: M6/ProyectoAlmacenamiento/app/src/main/java/com/example/proyectoalmacenamiento/MainActivity2.kt
package com.example.proyectoalmacenamiento

import android.content.Intent
import android.os.Bundle
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.proyectoalmacenamiento.databinding.ActivityMainBinding

class MainActivity2 : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnAceptar.setOnClickListener {
            val nombre = binding.txtNombre.text.toString()
            val apellido = binding.txtApellido.text.toString()
            val email = binding.txtEmail.text.toString()
            val celular = binding.txtCelular.text.toString()
            val sucursal = binding.txtSucursal.text.toString()

            val sp = getSharedPreferences("datos", MODE_PRIVATE)
            val editor = sp.edit()
            editor.putString("nombre", nombre).apply()
            editor.putString("apellido", apellido).apply()
            editor.putString("email", email).apply()
            editor.putString("celular", celular).apply()
            editor.putString("sucursal", sucursal).apply()

            Toast.makeText(this, "Datos guardados", Toast.LENGTH_SHORT).show()
        }

        binding.btnVer.setOnClickListener {
            val intent = Intent(this, VerActivity::class.java)
            startActivity(intent)
        }
    }
}